cloud/google/machineactuator.go

```diff
@@ -719,6 +705,28 @@ func (gce *GCEClient) getImagePath(img string) (imagePath string) {
 	return defaultImg
 }
 
+func newDisks(config *gceconfigv1.GCEProviderConfig, zone string, imagePath string, minDiskSizeGb int64) []*compute.AttachedDisk {
+	var disks []*compute.AttachedDisk
+	for idx, disk := range config.Disks {
+		diskSizeGb := disk.InitializeParams.DiskSizeGb
+		if diskSizeGb < minDiskSizeGb {
+			glog.Info("increasing disk size to %v gb, the supplied disk size of %v gb is below the minimum", minDiskSizeGb, diskSizeGb)
+			diskSizeGb = minDiskSizeGb
+		}
+		d := compute.AttachedDisk{
+			AutoDelete: true,
+			Boot:       idx == 0,
+			InitializeParams: &compute.AttachedDiskInitializeParams{
+				SourceImage: imagePath,
```

Comment:
I think image path is only for the boot disk.

Reply:
While all disks can have an image, this is a good catch in that the intention of this change was not to put the image on disks other than the boot disk! I'll change this and add a test for this scenario.